class/book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(date, person)
+    Rental.new(date, person, self)
+  end
```

Comment:
- [ ] @Chimoneg27 Great work implementing the `has-many ` side of the many-to-many (also has-many-through) relationship between Person and Book using the intermediate class Rental, however you forgot to do the same in your person class, kindly consider implementing it to achieve the many-to-many relationship

Reply:
Fixed.